### ISSUE TITLE
Auto Refresh Issue in Linux 

### ISSUE THREAD

**Issue by karimmd** [typora/typora-issues#3841]
The auto refresh does not work in Typora 0.9.95 on Linux. I am using the Keyboard shortcut Ctrl+Shift+P to do it manually. I have installed typora through deb package from the official repository. I am not sure whether anyone else having the same issue in their linux system. I am using KDE Neon 5.19 (Ubuntu Focal).

**Comment by abnerlee** [comment]
Do you mean auth refresh the file content?

Will it get refreshed when focus back to Typora?

**Comment by karimmd** [comment]
In the both system (KDE Neon OS), Typora is opened and the modified file is synced via dropbox. So if I have the same file opened in both of the system through typora then I need to manually press Ctrl+Shift+P to see the changes that I have made on the other system using Typora. But if I have different file opened in the other system then going to that file automatically receives the change that I made.

As compared to other writing apps like Sublime or VS code (through markdown extension), where I configured both of them to enable autosave and autoload. So if I made changes on the same file in one system, the other automatically changes while it is opened on the other system. 

I am not sure, whether I could make you to understand. 

In summary, if I make changes using typora in one system, it gets the updates on the other system but the changes are not visible while the file is opened in typora. 

But if I scroll from another file to that file then I can see the changes but if it is already opened inside typora then I need to press the Auto (Ctrl+Shift+P) to make it happen. And Typora does not even mention there is a change on that file. Even that would be nice to so that I would know there is a change on that file from another system. The Kwrite app in KDE does that.
